styleguide/source/assets/scss/02-molecules/_alert.scss

```diff
@@ -0,0 +1,58 @@
+.ama__alert {
+  background-color: $homepagePurple;
+  padding: $gutter / 2;
+  color: $white;
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  @include breakpoint($bp-med min-width) {
+    background-image: url('../icons/svg/icon-homepage-alert.svg');
```

Comment:
We probably can leverage the icons mixin here.
Please refer to _icons.scss

In a nutshell the icon name needs to be added and then you can reference the icon like this:

`<i class="icon--alert" />`

or `@extend icon--alert;`

Reply:
Right now, we aren't using `@extend icon--[icon]` anywhere else, but we are continuing to use icons as background images when they are decorative for larger breakpoints and hidden on smaller (see _resource-tabs.scss). Therefore, I'd rather leave this as is since it is consistent with that already implemented approach.